tests/pytests/test_config.py

```diff
@@ -94,7 +94,7 @@ def testAllConfig(env):
     env.assertEqual(res_dict['MAXAGGREGATERESULTS'][0], 'unlimited')
     env.assertEqual(res_dict['MAXEXPANSIONS'][0], '200')
     env.assertEqual(res_dict['MAXPREFIXEXPANSIONS'][0], '200')
-    env.assertIn(res_dict['TIMEOUT'][0], ['500', '100000'])
+    env.assertIn(res_dict['TIMEOUT'][0], ['500', '2147483647'])
```

Comment:
Not sure I follow those test changes... why was those changes needed?

Reply:
This change reflects the change in `tests/pytests/CMakeLists.txt`.

Reply:
So on config get it means that we will not get 0 but '2147483647'? Isn't it confusing?

Reply:
We can print `disabled` instead. WDYT?

Reply:
Sounds good, but why are we getting this number and not zero?

Reply:
Look at config.c 196-198.

Reply:
@ashtul why not return 0 ?

Reply:
@oshadmi Good idea. Implemented!